zerver/management/commands/runtornado.py

```diff
@@ -102,7 +102,7 @@ def inner_run():
             translation.activate(settings.LANGUAGE_CODE)
 
             print("Validating Django models.py...")
-            self.validate(display_num_errors=True)
+            self.check(display_num_errors=True)
```

Comment:
I guess it's https://docs.djangoproject.com/en/dev/releases/1.7/#modeladmin-validators, I'll just amend that into the commit message and merge.